docs/90-pkg-exporter

```diff
@@ -0,0 +1,9 @@
+# Apt hooks recommended for pkg-exporter
+
+# These hooks create files pre and post update, to compare the time difference inside the script
+APT::Update::Pre-Invoke {"touch /tmp/pkg-exporter-apt-update-pre"; };
+APT::Update::Post-Invoke {"touch /tmp/pkg-exporter-apt-update-post"; };
+
+# These hooks ensure the metrics are automatically updated when apt is run
+APT::Update::Post-Invoke-Success {"/usr/local/bin/pkg-exporter"};
```

Comment:
Until we provide a debian package, also those shall be a commented suggestion, as we do not yet have a single way of installation.
In my optionion starting a systemd unit (if present) would be better than calling the binary directly, however that needs some work on the systemd side.